src/app/confirm-dialog/confirm-dialog.component.html

```diff
@@ -0,0 +1,8 @@
+<h3 mat-dialog-title>{{data.title}}</h3>
+<mat-dialog-content>
+  {{data.content}}
+</mat-dialog-content>
+<mat-dialog-actions fxLayout="row" fxLayoutAlign="end center">
+  <button mat-button [mat-dialog-close]="false">Cancel</button>
```

Comment:
The button names here are kinda unclear (I clicked the "cancel" button inside this modal and wondered why I wasn't getting moved to the previous page). Maybe change the names from "Cancel" to "Continue Editing" and "OK" to "Leave Editor".

Reply:
Sounds good, I'll add that in quick